assets/js/googlesitekit/api/index.test.js

```diff
@@ -51,8 +51,7 @@ describe( 'googlesitekit.api', () => {
 	} );
 
 	afterEach( async () => {
-		fetchMock.restore();
-		fetchMock.mockClear();
+		fetchMock.reset();
```

Comment:
This should only be needed in `tests/js/setup-before-after.js`, similar to how the previous `fetch` mock was used.

Reply:
:+1: